### DISCUSSION TITLE
wlan1 dont get it

### DISCUSSION THREAD

**Discussion by c1berseguro** [martin-olivier/airgorah#96] [General]
Hi, thanks for the info, the only thing is that EVEN IF you have wlan0 for your normal network and wlan1 for your extra network card, the tool takes wlan0 as default. even if you select it at the beginning. It also happens with the FernFire Cracker tool. do you know how to solve it, thanks.

**Comment by martin-olivier** [comment]
Hello @c1berseguro,

The network interface selection is working correctly.
Can you check using `ifconfig` before and after interface selection ?

**Reply by c1berseguro** [reply]
thanks oliver, the interface work well done.
Your knowledge is power. Thanks.

I will try BruteForce in Gnome because dont show anything when u execute this module with file.cap and rockyou.txt. See you soon

**Reply by martin-olivier** [reply]
Hello @c1berseguro,

This issue with bruteforce and gnome terminal has been fixed in the latest version (v0.7.1)
https://github.com/martin-olivier/airgorah/releases/tag/v0.7.1

**Comment by john1051** [comment]
Hello, could you please help me with this error when trying to install it on this version of Kali?

VERSION_ID="2025.2"
VERSION="2025.2"
All packages are up to date.
You might want to run apt --fix-broken install to correct these.
Unsatisfied dependencies:

airgorah : Depends: policykit-1 but it is not installable
Depends: mdk4 but it is not installed
Error: Unmet dependencies. Try apt --fix-broken install with no packages (or specify a solution).
